cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -174,14 +189,14 @@ func (m command) validateFlags(c *cli.Context, config *config) error {
 	config.mode = "config-file"
 
 	switch config.runtime {
-	case "containerd", "crio", "docker":
+	case runtimeContainerd, runtimeCrio, runtimeDocker:
```

Comment:
I would prefer to make these kinds of changes in a separate commit to make the actual additions here easier to review.

Reply:
I'm fine to keep these in for now.

Reply:
Sure